### PR TITLE
Adding teams info

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/10169

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsPrivacyConfiguration.md [hunk 1]
```diff
@@ -178,6 +178,7 @@ In advanced privacy mode, only people on your Contacts list will be allowed to v
 If False, your presence information will be available to anyone in your organization.
 The default value is False.
 
+For information regarding privacy mode in Microsoft Teams, see [User presence in Teams](/microsoftteams/presence-admins)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsPrivacyConfiguration.md:181]:
```suggestion
For information regarding privacy mode in Microsoft Teams, see [User presence in Teams](/microsoftteams/presence-admins).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this Pr. Thanks

File: skype/skype-ps/skype/Set-CsPrivacyConfiguration.md [hunk 2]
```diff
@@ -178,6 +178,7 @@ In advanced privacy mode, only people on your Contacts list will be allowed to v
 If False, your presence information will be available to anyone in your organization.
 The default value is False.
 
+For information regarding privacy mode in Microsoft Teams, see [User presence in Teams](/microsoftteams/presence-admins).
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsPrivacyConfiguration.md:181]:
```suggestion
For information about privacy mode in Microsoft Teams, see [User presence in Teams](/microsoftteams/presence-admins).
```

**Pull request merged**